xdg-autostart: Fall back to default XDG variable values

Konfekt opened the pull request:
For example, $XDG_RUNTIME_DIR might be unset in Debian 12

Konfekt committed:
xdg-autostart: Fall back to default XDG variable values

For example, $XDG_RUNTIME_DIR might be unset in Debian 12

lcpz commented on rc.lua.template:
```diff
@@ -76,7 +76,8 @@ awful.spawn.with_shell(
     -- list each of your autostart commands, followed by ; inside single quotes, followed by ..
-    'dex --environment Awesome --autostart --search-paths "$XDG_CONFIG_DIRS/autostart:$XDG_CONFIG_HOME/autostart"' -- https://github.com/jceb/dex
+    'dex --environment Awesome --autostart --search-paths ' ..
+    '"${XDG_CONFIG_HOME:-$HOME/.config}/autostart:${XDG_CONFIG_DIRS:-/etc/xdg}/autostart";') -- https://github.com/jceb/dex
```
Are you sure that final `)` is not redundant?

Konfekt replied:
It is. I forgot about the spurious `)` in line 81

Konfekt committed:
remove spurious parenthesis 

as observered by @lcpz in https://github.com/lcpz/awesome-copycats/pull/313/files#r1797683276
